homeassistant/components/wemo/entity.py

```diff
@@ -133,6 +140,14 @@ def device_info(self) -> Dict[str, Any]:
             "manufacturer": "Belkin",
         }
 
+    @property
+    def capability_attributes(self) -> Dict[str, Any]:
```

Comment:
We shouldn't override this. It's implemented by component base class only. What's the intended use case?

Reply:
Not all WeMo devices within a platform support this capability/feature. Only the wall-mounted switches support long-press actions when the button is held down. Plug-in switches do not support this feature.

I was unsure of how to limit the device trigger so it only applies to devices that support long-press actions: https://github.com/home-assistant/core/pull/45503/files#diff-038dae26a83fbd97d3747def6980752c56a6774f84784262aef3886b05076b54R35
Is there a better way to do this?

Reply:
Look at eg the deconz integration device trigger module.

Reply:
We should try to get access to the wemo device instance by looking it up somehow. Entities should probably not be involved at all.

Reply:
I've removed the overridden `capability_attributes` method.